pom.xml

```diff
@@ -0,0 +1,19 @@
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.mozilla.telemetry</groupId>
+    <artifactId>mozilla-pipeline-schemas</artifactId>
+    <version>0.1-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>com.github.everit-org.json-schema</groupId>
+            <artifactId>org.everit.json.schema</artifactId>
+            <version>1.12.1</version>
+        </dependency>
+    </dependencies>
```

Comment:
We should probably also specify the same explicit version of org.json that we're using in ingestion-beam.

Reply:
This looks to be the case. I've added to a comment pointing to pom.xml in gcp-ingestion to be clear about this.

Reply:
Understood, I misread and took `org.json` to be the same thing as `org.everit.json.schema`.